SU2_CFD/src/output/output_structure_legacy.cpp

```diff
@@ -37,9 +37,9 @@ COutputLegacy::COutputLegacy(CConfig *config) {
   size = SU2_MPI::GetSize();
 
   unsigned short iDim, iZone, iSpan, iMarker;
-  
+
   /*--- Initialize point and connectivity counters to zero. ---*/
```

Comment:
It seems I accidentally removed the trailing spaces in this file (output_structure_legacy.cpp)... If that inconveniences anyone let me know.